mods/religion/init.lua

```diff
@@ -0,0 +1,117 @@
+-- religion/init.lua
+
+-- Minetest mod: religion
+-- See README.txt for licensing and other information.
+
+-- Load support for game translation.
+local S = minetest.get_translator("religion")
+
+-- time in seconds before getting a responce from god
+local pray_time = 2 
+
+-- the higher the weight, higher is the probability of getting it 
+local pray_responses = {
+	{message="and nothing happens", weight=100},
+	{message="and you feel refreshed", weight=20},
+	{message="you feel how your faith is strengthened", weight=20},
+	{message="something inside you tells you that this is not working", weight=10},
+	{message="and you feel a soft breeze that comforts you", weight=10},
+}
```

Comment:
Here we can add more messages, and each one have it's own probability (weight) to happen.